subtests/docker_cli/create/create_tmpfs.py

```diff
@@ -0,0 +1,28 @@
+"""
+Create container with --tmpfs option and inspect it
+"""
+
+from create import create_base
+from dockertest.output import DockerVersion
+
+
+class create_tmpfs(create_base):
+
+    def initialize(self):
+        super(create_tmpfs, self).initialize()
+        # Minimum docker version 1.10 is required for --tmpfs
+        DockerVersion().require_server("1.10")
+
+    def run_once(self):
+        super(create_tmpfs, self).run_once()
+        cid = self.get_cid()
+        cont_long_id = self.sub_stuff['cont'].json_by_long_id(cid)
```

Comment:
Using a temporary variable to avoid line wrap is fine. My question was about the variable's name: `cont_long_id` is a weird name. To me it doesn't indicate the variable's purpose or content. I actually find it more confusing than `x` or even `foo`: I see the name `cont_long_id` and my first thought is that it contains an id, or a container, or at least _something_ relating to `cont` or `long` or `id`. But AFAICT it doesn't, which means it will confuse the next maintenance programmer too.